install/operator/deploy/role_binding.yaml

```diff
@@ -1,11 +1,16 @@
 kind: RoleBinding
-apiVersion: rbac.authorization.k8s.io/v1
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Should this be?

```suggestion
apiVersion: rbac.authorization.k8s.io/v1
```

Reply:
you probably right, I blindly copy from https://github.com/syndesisio/syndesis/blob/e14633aa7e591f635a9466ba4e28e6b5fa66557a/install/operator/pkg/generator/assets/install/operator.yml.tmpl#L13, maybe @chirino can shed some lights